### ISSUE TITLE
popup is not keyboard focusable / cannot close with keyboard

### ISSUE THREAD

**Issue by tonyhallett** [PixiEditor/ColorPicker#28]

**Comment by Equbuxu** [comment]
After some googling it seems like this is a quirk of WPF, a fix would require making an attached behavior that sets the focus manually. I'll ignore this for now but it's definitely something that will need to be fixed eventually

**Comment by tonyhallett** [comment]
Agreed. Quirk is one word for it. I have a couple of behaviours that do the trick. Handle the opened event and focus accordingly.

**Comment by flabbet** [comment]
Closing due to inactivity
